doc/nrf/ug_radio_fem.rst

```diff
@@ -191,3 +191,7 @@ Then there is no need to define a GPIO to control the PDN signal. The line ``pdn
 
 Generally, if pin ``X`` is not used, the ``X-gpios = < .. >;`` property can be removed.
 This applies to all properties with a ``-gpios`` suffix, for both nRF21540 and SKY66112-11.
+
+.. _ug_radio_fem_nrf21540_ek:
```

Comment:
@KAGA164 this is something we don't still see in the doc. Have a section "Supported boards" mentioning DK and EK and then nRF21450 EK under that. Otherwise, the EK story comes suddenly without any explanation.

Reply:
Sure, I can add a commit for this :)

Reply:
@peknis I squashed Mia's commit. I hope that doc is good enough now :)